tests/test_obslog_command.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::path::Path;
+
+use common::TestEnvironment;
+use regex::Regex;
+
+pub mod common;
+
+#[test]
+fn test_obslog_with_or_without_diff() {
```

Comment:
How about the fold case, where you have two predecessors? (or any other case where the number of parents changes)

Reply:
I added a test for the fold/squash case a bit further down